ods_ci/tests/Resources/Page/HybridCloudConsole/HCCLogin.robot

```diff
@@ -9,8 +9,8 @@ Is SSO Login Page Visible
   RETURN  ${login_prompt_visible}
 
 Wait For HCC Splash Page
-   Wait Until Page Contains Element    xpath://span[contains(@class, 'pf-c-spinner')]   timeout=15
-   Wait Until Page Does Not Contain Element    xpath://span[contains(@class, 'pf-c-spinner')]   timeout=20
+   Wait Until Page Contains Element    xpath://span[contains(@class, 'pf-v5-c-spinner')]   timeout=15
+   Wait Until Page Does Not Contain Element    xpath://span[contains(@class, 'pf-v5-c-spinner')]   timeout=20
```

Comment:
hm probably we could use `-c-spinner` only. My concern is that `-v5-` is supposed to change over versions from now on

Reply:
This one was a quick fix, shouldn't we invest more time on it when we have data-testid field ready and replace xpaths then to be not dependant on it. Because this -c-spinner approach will only work on 'contains' xpaths not all.

Reply:
>when we have data-testid field ready


is it ready? if yes, then we should replace as you're suggesting, sooner than later. If we leave "v5", I can see these tests broken some time in the future.

Ok with this PR if it's intended to be temporary